assets/src/components/routeLadder.tsx

```diff
@@ -91,7 +91,17 @@ export const NewHeader = ({
   return (
     <Card className="c-new-route-ladder__header">
       <Card.Body>
-        <div className="c-new-route-ladder__fake-dropdown" />
+        {inTestGroup(TestGroups.DetoursPilot) && (
+          <Dropdown>
+            <Dropdown.Toggle className="c-route-ladder__dropdown-button"></Dropdown.Toggle>
+            <Dropdown.Menu className="c-route-ladder__dropdown">
+              <Dropdown.Item>
+                <PlusSquare className="c-route-ladder__dropdown-item__icon" />
+                Add detour
+              </Dropdown.Item>
+            </Dropdown.Menu>
+          </Dropdown>
```

Comment:
suggestion: Most of this styling can be done with bootstrap

```suggestion
          <Dropdown className="border-box inherit-box">
            <Dropdown.Toggle style={{aspectRatio: "1/1"}}/>
            <Dropdown.Menu>
              <Dropdown.Item className="icon-link">
                <PlusSquare />
                Add detour
              </Dropdown.Item>
            </Dropdown.Menu>
          </Dropdown>
```

Though you may want to keep `className="c-route-ladder__dropdown-button"` and put the `aspect-ratio` on that class.

```suggestion
          <Dropdown className="border-box inherit-box">
            <Dropdown.Toggle className="c-route-ladder__dropdown-button"/>
            <Dropdown.Menu>
              <Dropdown.Item className="icon-link">
                <PlusSquare />
                Add detour
              </Dropdown.Item>
            </Dropdown.Menu>
          </Dropdown>
```

Reply:
Oh for sure, so first I want to caveat that `aspect-ratio` is not an attribute I tend to use, since usually I'm building something I do _not_ want to scale. When I tried `aspect-ratio` in this case, no effect; it only works if I set the height I want to use to `36px`. Then, since width is auto, the `aspect-ratio` is correctly applied and width expands to be `36px` as well. 

I'm not sure why the height is required here, because from the docs, I'd expect aspect-ratio to compromise between height and width if they are both set to auto. It seems to me that height is not auto, but inherited from somewhere. So in that case, if I need to set both height and aspect-ratio, then I might as well just set height and width.